frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -227,6 +229,14 @@ export const EditZing = () => {
 
   return courseInfo && hasLoadedStudentData ? (
     <StyledContainer>
+      {isEmailing ? (
```

Comment:
Entrypoint 1 for connecting to new button + edit

Reply:
Instead of
```js
{isEmailing ? <...> : null}
```
You could use
```js
{isEmailing && <...>}
```